src/libs/terms-of-service-alerts.ts

```diff
@@ -1,19 +1,21 @@
-import { atom } from '@terra-ui-packages/core-utils';
+import { Atom, atom } from '@terra-ui-packages/core-utils';
 import _ from 'lodash/fp';
-import { Fragment, useEffect } from 'react';
+import { Fragment, ReactNode, useEffect } from 'react';
 import { br, h } from 'react-hyperscript-helpers';
 import { Link } from 'src/components/common';
 import { Ajax } from 'src/libs/ajax';
 import * as Nav from 'src/libs/nav';
 import { useStore } from 'src/libs/react-utils';
-import { authStore } from 'src/libs/state';
+import { AuthState, authStore, TermsOfServiceStatus } from 'src/libs/state';
 
-const getNewTermsOfServiceNeedsAcceptingAlert = async (termsOfServiceState) => {
-  const shouldNotify = !termsOfServiceState.userHasAcceptedLatestTos && termsOfServiceState.permitsSystemUsage;
-  if (!shouldNotify) {
-    return null;
-  }
+type TermsOfServiceAlert = {
+  id: string;
+  title: string;
+  message: ReactNode;
+  severity: string;
+};
 
+const getNewTermsOfServiceNeedsAcceptingAlert = async (): Promise<TermsOfServiceAlert | null> => {
```

Comment:
It looks like this always returns an Alert.

```suggestion
const getNewTermsOfServiceNeedsAcceptingAlert = async (): Promise<TermsOfServiceAlert> => {
```

Reply:
Oops, I refactored it specifically to get rid of the null type there, but then I forgot to update the type.